src/person/actions/update-address.action.ts

```diff
@@ -20,8 +21,20 @@ export const updateAddress = async (
         throw errorFactory.create(PersonErr.PUT_ADDRESS_FAILED);
       });
 
-    return response.data;
+    switch (response.status) {
+      case 200:
+        return response.data;
+      case 403:
+        throw errorFactory.create(PersonErr.PUT_ADDRESS_FORBIDDEN);
+      case 404:
+        throw errorFactory.create(PersonErr.PUT_ADDRESS_NOT_FOUND);
```

Comment:
Plutôt faire uniquement référence à la ressource pour les erreurs 404.

```suggestion
        throw errorFactory.create(PersonErr.ADDRESS_NOT_FOUND);
```